docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+## Error handling
+
+### Aborting with unhandled error
```

Comment:
As we discussed in Discord: I think we can skip this recipe as it merely shows the default situation anyway ie. a script without error handling. It's better to show a recipe that actually includes error handling (which I guess is the next one), so the user can copy the approach. It's also a bit odd to me that the section is called "Error handling" but the first recipe is "abort with unhandled error".